crypto/crypto/src/traits.rs

```diff
@@ -24,6 +24,8 @@ use thiserror::Error;
 #[derive(Clone, Debug, PartialEq, Eq, Error)]
 #[error("{:?}", self)]
 pub enum CryptoMaterialError {
+    /// Struct to be signed does not serialize correctly.
```

Comment:
This should never happen.

Reply:
It can still happen if for example we try to serialize an array of 2^31+ elements. Let’s revisit that in an upcoming PR.

Reply:
See https://github.com/libra/libra/pull/4073#discussion_r433229305